### ISSUE TITLE
false positive SA1008

### ISSUE THREAD

**Issue by msmolka** [DotNetAnalyzers/StyleCopAnalyzers#3376]
Following code causing false positive  SA1008:

```c#
await (await repository.GetQueryable()).ToListAsync();
```

**Comment by bjornhellander** [comment]
Hi! I tried to reproduce this using some simple similar code, but failed. What version are you using? Where exactly is the diagnostic shown?

**Comment by msmolka** [comment]
Looks working in1.2.0-beta.354. Closing...
